packages/gensx/src/utils/bundler.ts

```diff
@@ -68,9 +86,54 @@ export async function bundleWorkflow(
     // This is handled by the build.sh script
     return path.join(outDir, "dist.tar.gz");
   } catch (error) {
-    console.error("Error bundling workflow", error);
+    console.error("Error bundling workflow with Docker", error);
     console.error("Stdout:\n", stdout);
     console.error("Stderr:\n", stderr);
     throw error;
   }
 }
+
+async function bundleLocally(
+  packageJsonDir: string,
+  relativeWorkflowPath: string,
+  outDir: string,
+): Promise<string> {
+  try {
+    // Create dist directory inside outDir
+    const distDir = path.join(outDir, "dist");
+    mkdirSync(distDir, { recursive: true });
+
+    // Move to the project directory
+    const originalDir = process.cwd();
+    process.chdir(packageJsonDir);
+
+    // Install @vercel/ncc if not already installed
+    try {
+      await execPromise("npx --no-install @vercel/ncc --version");
+    } catch (_error) {
+      await execPromise("npm install -g @vercel/ncc");
+    }
+
+    // Build with ncc
+    const { stdout: _stdout, stderr } = await execPromise(
+      `npx @vercel/ncc build ./${relativeWorkflowPath} -o ${distDir} --target es2022`,
+    );
+
+    if (stderr) {
+      // Process stderr if needed
+    }
+
+    // Create tar.gz file (for compatibility with Docker build)
+    process.chdir(distDir);
+    await execPromise(`tar -czvf ../dist.tar.gz *`);
```

Comment:
Is this used for anything?

Reply:
Not really; just makes it more like the docker build step.  i'll remove it.

Reply:
removed the extra code here